packages/sources/coinmetrics/src/endpoint/lwba-ws.ts

```diff
@@ -89,16 +89,14 @@ type WsPairQuoteMessage =
   | WsCryptoLwbaErrorResponse
   | WsCryptoLwbaReorgResponse
 
-export const calculatPairQuotesUrl = (
+export const calculatePairQuotesUrl = (
   context: EndpointContext<WsCryptoLwbaEndpointTypes>,
   desiredSubs: (typeof inputParameters.validated)[],
 ): string => {
   const { API_KEY, WS_API_ENDPOINT } = context.adapterSettings
-  const pairs = [
-    ...new Set(desiredSubs.map((sub) => `${sub.base.toLowerCase()}-${sub.quote.toLowerCase()}`)),
-  ].join(',')
-  const generated = new URL('/v4/timeseries-stream/pair-quotes', WS_API_ENDPOINT)
-  generated.searchParams.append('pairs', pairs)
+  const assets = [...new Set(desiredSubs.map((pair) => pair.base.toLowerCase()))].join(',')
```

Comment:
Sorry to jump in here after this was approved but could we sort assets here like we do in `price-ws`? The framework will reestablish the ws connection if it notices a URL change. If this isn't sorted, there's a chance the fw keeps reestablishing connection for what could be the same URL.
```suggestion
  const assets = [...new Set(desiredSubs.map((pair) => pair.base.toLowerCase()))].sort().join(',')
```